Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -86,28 +85,37 @@ public void Initialize()
             m_TouchState.Reset();
             m_SeenTouchEvents = false;
 
-            m_Cfg = Configuration.GetDefaultConfiguration();
-
+            SelectInputActionAsset();
             RegisterActions();
 
+            // TODO make it configurable as it is not part of default config
```

Comment:
Is this TODO intended to remain, if that is the case I wonder if it should have a ticket filed?

Reply:
I just copied it over to not lose any information written by a previous author. Happy to create a Jira ticket and remove the comment if preferred - whatever you think is better?

Reply:
If it was already there there to begin with I do no think it needs to prevent this PR from progressing.